pkg/component/networking/istio/charts/istio/istio-ingress/templates/metrics-serviceentry.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: networking.istio.io/v1
+kind: ServiceEntry
+metadata:
+  name: metrics
+  namespace: {{ .Release.Namespace }}
+spec:
+  exportTo:
+  - '*'
+  hosts:
+  - metrics.istio.local
+  location: MESH_INTERNAL
+  ports:
+  - name: metrics
+    number: 15020
+    protocol: HTTP
+  resolution: DNS
```

```suggestion
  resolution: STATIC
  endpoints:
  - address: 127.0.0.1
```
Would it work if the `ServiceEntry` is configured like this and we don't add the host alias to the istio pods?

---

I only tried with `addressed` instead of `endpoints`. With `addresses` it did not work, but `endpoints` seems to work so that we do not need to adapt the deployments.
Great finding.